en/debug/gdb_debugging.md

```diff
@@ -54,13 +54,13 @@ Enter this depending on your system:
 
 ##### Fedora:
 ```bash
-env LD_PRELOAD=/lib64/libtcmalloc.so ../src/firmware/posix/px4 ../../posix-configs/SITL/init/lpe/iris
+env LD_PRELOAD=/lib64/libtcmalloc.so ../src/firmware/posix/px4 -s ../../posix-configs/SITL/init/lpe/iris
```

Comment:
I was a bit suspicious that commands were outdated in other ways beside what I was fixing. Implemented your fix in 3b7bb69 . Are you sure it isn't init.d-posix/rcS instead of etc/init.d-posix/rcS ? At least that's where it was for me.

Reply:
Yes, `etc` is required. In `rootfs` you'll have a symlink `etc` to ROMFS/px4fmu_common.

Reply:
Ok, from the description didn't expect the rootfs_directory argument to work the way it works. I thought that rootfs_directory needs to point to directory containing etc directory and behaves more like cd to rootfs_directory or prepending it to all relative paths.

Reply:
@bkueng  Personally I don't understand any of the logic for any of what is described above - it is all just turnkey instructions with no context. That kind of sux, because (in a case like this) if something goes wrong there is nothing to be done except raise an error. 

But since you do, can you please check whether the description referred to in https://github.com/PX4/Devguide/pull/708#discussion_r246332358 is ambigous and update as appropriate?

Merge whenever you are happy.

Reply:
@karliss It's not quite clear to me how you would like to see it being updated, but please do the required changes as you see fit (https://github.com/PX4/Firmware/blob/master/platforms/posix/src/main.cpp#L557).

I updated the path and merged it.